x11-misc/polybar/polybar-3.1.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+XPP_VERSION="1.4.0"
+I3IPCPP_VERSION="0.7.1"
+
+DESCRIPTION="A fast and easy-to-use tool for creating status bars"
+HOMEPAGE="https://github.com/jaagr/polybar"
+SRC_URI="https://github.com/jaagr/polybar/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/jaagr/xpp/archive/${XPP_VERSION}.tar.gz
+	https://github.com/jaagr/i3ipcpp/archive/v${I3IPCPP_VERSION}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="alsa curl i3wm ipc mpd network"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="x11-base/xcb-proto
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-image
+	x11-libs/libxcb
+	alsa? ( media-libs/alsa-lib )
+	i3wm? ( dev-libs/jsoncpp )
+	mpd? ( media-libs/libmpdclient )
+	curl? ( net-misc/curl )
+	network? ( net-wireless/wireless-tools )
+"
+
+DEPEND="${PYTHON_DEPS}
+	dev-lang/python:2.7
```

Comment:
python dep can be dropped as it is handled by python*eclass. As this change leads to DEPEND="${RDEPEND}", exchange the variables so that DEPEND contains all deps and RDEPEND="${DEPEND}"